### ISSUE TITLE
Calibrar o computador com o carro

### ISSUE THREAD

**Issue by gmfc** [gmfc/arduinoLab#10]

**Comment by skewerr** [comment]
Você quer dizer com leitura de sensores ou...?

**Comment by gmfc** [comment]
Sim, exemplo, se o sensor de RPM da roda me manda uma variação de 1.2 volts, isso quer dizer uma variação de RPM X. Preciso ver essa relação com todos os sensores, tanto em entrada quanto saída.
